caravel/views.py

```diff
@@ -2124,30 +2124,39 @@ def queries(self, last_updated_ms):
     def search_queries(self):
         """Search for queries."""
         query = db.session.query(models.Query)
-        userId = request.args.get('userId')
-        databaseId = request.args.get('databaseId')
-        searchText = request.args.get('searchText')
+        user_id = request.args.get('userId')
+        database_id = request.args.get('databaseId')
```

Comment:
s/.get('databaseId')/.get('database_id') - to be consistent
for now we use python conversion for args